aws/cmd/functions.go

```diff
@@ -34,6 +34,16 @@ var (
 			// Returned values are commented in the interface doc comment block.
 			`,
 		},
+		Function{
+			Entity:  "VpcPeeringConnections",
```

Comment:
Should it maybe be named `VPC..`?

Reply:
:+1: 

I took the format from https://github.com/cycloidio/terracognita/blob/master/aws/cmd/functions.go#L29

Should we change both to VPC ?

Reply:
Ok nothing, this is the one used on their side, it's ok :).